### PR TITLE
fix: closed method unit test

### PR DESCRIPTION
**Pull request by summer-ji-eng**
address [issue](https://github.com/googleapis/nodejs-bigquery-storage/runs/5457807003?check_suite_focus=true)

initialize the request with sub field value. Also fix the unused variable in linter issue

### CODE REVIEW

File: baselines/disable-packing-test/test/gapic_echo_v1beta1.ts.baseline [hunk 1]
```diff
@@ -541,8 +541,7 @@ describe('v1beta1.EchoClient', () => {
               projectId: 'bogus',
         });
             client.initialize();
-            const request = generateSampleMessage(new protos.google.showcase.v1beta1.ExpandRequest());
-            const expectedError = new Error('The client has already been closed.');
+            const request = generateSampleMessage(new protos.google.showcase.v1beta1.ExpandRequest());const expectedError = new Error('The client has already been closed.');
```
Comment (alexander-fenster) [inline baselines/disable-packing-test/test/gapic_echo_v1beta1.ts.baseline:544]: nit: linebreak is needed here

**Review by alexander-fenster — APPROVED** [review]
Please fix the linebreaks, I believe you're eating up an extra newline by using `-%}` instead of `%}` somewhere.

File: baselines/bigquery-storage/test/gapic_big_query_storage_v1beta1.ts.baseline [hunk 2]
```diff
@@ -640,7 +636,9 @@ describe('v1beta1.BigQueryStorageClient', () => {
         });
             client.initialize();
             const request = generateSampleMessage(new protos.google.cloud.bigquery.storage.v1beta1.ReadRowsRequest());
-            const expectedError = new Error('The client has already been closed.');
+            request.readPosition = {};
+            request.readPosition.stream = {};
+            request.readPosition.stream.name = '';const expectedError = new Error('The client has already been closed.');
```
[the remark this answers is not in the document]
Comment (summer-ji-eng) [reply baselines/bigquery-storage/test/gapic_big_query_storage_v1beta1.ts.baseline]: SG

**Comment by summer-ji-eng** [comment]
> Please fix the linebreaks, I believe you're eating up an extra newline by using `-%}` instead of `%}` somewhere.

Thank you. You are much faster than my updated PR submitting.

**Pull request merged**